src/braket/circuits/circuit.py

```diff
@@ -196,6 +199,27 @@ def qubits(self) -> QubitSet:
         """QubitSet: Get a copy of the qubits for this circuit."""
         return QubitSet(self._moments.qubits.union(self._qubit_observable_set))
 
+    @property
+    def is_parameterized(self) -> bool:
```

Comment:
I think this name is ambiguous; it's a mouthful, but maybe something like `has_free_parameters`